Sources/Widgets/Widgets3D/LineWidget/helper.js

```diff
@@ -16,3 +18,18 @@ export function updateTextPosition(model) {
   const SVGTextState = model.widgetState.getText();
   SVGTextState.setOrigin(calculateTextPosition(model));
 }
+
+export function getNbHandles(model) {
+  const handle1 = model.widgetState.getHandle1();
+  const handle2 = model.widgetState.getHandle2();
+  const moveHandle = model.widgetState.getMoveHandle();
+  if (handle1.getOrigin().length !== 0 && handle2.getOrigin().length !== 0) {
```

Comment:
isn't the following working ?
```
if (handle1.getOrigin() && handle2.getOrigin())
```

Reply:
No, since the value of the tab is set [here](https://github.com/ocrossi/vtk-js/blob/a6ae8abd3b3073e664157a7ef3caaa7e2111ff4a/Sources/Widgets/Widgets3D/LineWidget/state.js#L22), you can't check if points are placed this way. Length on the other hand is ok